<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>830</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../page2resource.qrc">:/back/배경화면 그룹.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>742</y>
     <width>47</width>
     <height>57</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/홈버튼 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>753</y>
     <width>33</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/뒤로가기 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tip_btn">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>748</y>
     <width>44</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/팁버튼.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ai_btn">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>294</y>
     <width>171</width>
     <height>348</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/ai trainer 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="free_btn">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>294</y>
     <width>171</width>
     <height>348</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/자유운동하기 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>-34</x>
     <y>51</y>
     <width>133</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/로그아웃 버튼.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../page2resource.qrc"/>
 </resources>
 <connections/>
</ui>
